### PR TITLE
Fix claim assessment timelock

### PR DESCRIPTION
**Pull request by shark0der**

### CODE REVIEW

**Comment by danoctavian** [comment]
worth having a unit/integration test for this new logic

File: contracts/modules/claims/ClaimsReward.sol [hunk 1]
```diff
@@ -529,10 +528,25 @@ contract ClaimsReward is IClaimsReward, LegacyMasterAware {
...
+      uint lockTimePerVote = td.lockCADays();
+      uint reduction = lockTimePerVote.mul(counter);
+      uint minExpiration = block.timestamp.add(lockTimePerVote);
+      uint currentExpiration = tc.getLockedTokensValidity(msg.sender, "CLA");
+
+      // reduce lock time only if current lock time is greater than min lock time
+      if (currentExpiration > minExpiration) {
+
+        // and never reduce to less than min lock time
+        if (currentExpiration.sub(reduction) < minExpiration) {
```
Comment (danoctavian) [inline contracts/modules/claims/ClaimsReward.sol:542]: so this means that you will always have to wait at least min lock time correct?

There's no way to get this lock time down to block.timestamp

File: contracts/modules/claims/ClaimsReward.sol [hunk 2]
```diff
@@ -529,10 +528,25 @@ contract ClaimsReward is IClaimsReward, LegacyMasterAware {
...
+          reduction = currentExpiration - minExpiration;
```
Comment (danoctavian) [inline contracts/modules/claims/ClaimsReward.sol:543]: use .sub here

**Review by roxdanila — CHANGES_REQUESTED** [review]
Fix looks good.
Please add a test as discussed offline and then loop in Iosiro for the final review.

**Review by roxdanila — APPROVED** [review]

**Pull request merged**